src/openrct2-ui/windows/TitleLogo.cpp

```diff
@@ -16,16 +16,15 @@
 
 namespace OpenRCT2::Ui::Windows
 {
-    static constexpr int32_t WW = 232;
-    static constexpr int32_t WH = 136;
+    static constexpr ScreenSize kWindowSize = { 232, 136 };
 
     enum
     {
         WIDX_LOGO
     };
 
     static constexpr auto _titleLogoWidgets = makeWidgets(
-        makeWidget({ 0, 0 }, { WW + 1, WH + 1 }, WidgetType::imgBtn, WindowColour::primary));
+        makeWidget({ 0, 0 }, { kWindowSize.width + 1, kWindowSize.height + 1 }, WidgetType::imgBtn, WindowColour::primary));
```

Comment:
I don't understand this. Widgets should never go beyond their window sizes. Seems like a historical mistake?